events/events.go

```diff
@@ -74,13 +74,59 @@ func EmitInitComplete(nodeID types.NodeID) {
 	)
 }
 
+func EmitRegisteredInPoet(url string, challenge []byte, roundID string) {
+	const help = "Registered in PoET."
+	emitUserEvent(
+		help,
+		false,
+		&pb.Event_RegisteredInPoet{
+			RegisteredInPoet: &pb.EventRegisteredInPoet{
+				Url:       url,
+				Challenge: challenge,
+				RoundId:   roundID,
+			},
+		},
+	)
+}
+
+func EmitProofDownloadedFromPoet(url, roundID string, ticks uint64) {
+	const help = "Downloaded proof from PoET."
+	emitUserEvent(
+		help,
+		false,
+		&pb.Event_ProofDownloadedFromPoet{
+			ProofDownloadedFromPoet: &pb.EventProofDownloadedFromPoet{
+				Url:     url,
+				Ticks:   ticks,
+				RoundId: roundID,
+			},
+		},
+	)
+}
+
+func EmitBestProofSelected(url, roundID string, ticks uint64) {
+	const help = "Best PoET proof selected."
+	emitUserEvent(
+		help,
+		false,
+		&pb.Event_BestProofSelected{
+			BestProofSelected: &pb.EventBestProofSelected{
+				Url:     url,
+				Ticks:   ticks,
+				RoundId: roundID,
+			},
+		},
+	)
+}
```

Comment:
I think this also needs a `smesher` field. The best proof might be different for different identities (i.e. if one identity failed to register at one of the PoETs)